### DISCUSSION TITLE
Support for chartjs-rc

### DISCUSSION THREAD

**Discussion by boddunan** [sgratzl/chartjs-chart-geo#55] [General]
The plugin no longer works with latest version of chartjs. Also the codepen examples do not work.

https://codepen.io/sgratzl/pen/bGVmQKw

Can you please let me know how to fix this?

**Comment by sgratzl** [comment]
should be fixed with https://github.com/sgratzl/chartjs-chart-geo/releases/tag/v3.0.0-rc.0
